src/moin/static/css/common.css

```diff
@@ -689,6 +689,30 @@ ul#moin-rev-navigation { text-align: center; background-color: var(--bg-zebra);
 .moin-rst-optionlist td:first-child,
 .moin-rst-fieldlist td:first-child { font-weight: bold; }
 
+/* MonthCalendar macro */
+
+.calendar a:link, .calendar a:visited { color: #202020; color: var(--cal-text); text-align: center; }
+.calendar table,
+.calendar th,
+.calendar thead,
+.calendar tr { padding: 0; border: 3px solid #FCFCFC; border: 3px solid var(--border-cal); text-align: center; }
```

Comment:
Sorry, https://github.com/moinwiki/moin/issues/1193 drop support for IE11 was recently closed, so all the duplicate border and color attributes should be removed

border: 3px solid #FCFCFC; border: 3px solid var(--border-cal);

becomes

border: 3px solid var(--border-cal);

Reply:
Many thanks for checking. Your reported issues are fixed.